modules/scheduler/scheduler/src/test/java/org/motechproject/scheduler/it/MotechSchedulerServiceImplBundleIT.java

```diff
@@ -117,6 +124,197 @@ public void shouldScheduleCronJob() throws SchedulerException {
     }
 
     @Test
+    public void shouldScheduleCronJobFromDto() throws Exception {
```

Comment:
If we use a different component for translation between the scheduler services and dto's, then it will only require a unit test testing the mapping from the dto to scheduler calls. Integration tests are generally more expensive and can be avoided here.